Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverInformation.yaml

```
@@ -358,4 +358,6 @@ DriverInformation:
       kvFunction: updateOneWithKV
       params: [servicesEnabledForSubscription]
       where: driverId
+    updateServicesEnabledForSubscription:
```

YAMLlint (1.35.1)

[error] duplication of key "updateServicesEnabledForSubscription" in mapping

(key-duplicates)